views/restaurants.pug

```diff
@@ -52,12 +59,19 @@ block content
                 thead
                     tr
                         th Restaurant Names
+                        th Cuisine
 
                 tbody
                     each restaurant in restaurants
                         tr
                             td
                                 a(href="/restaurants/" + restaurant.rname)= restaurant.rname
+                            td
+                                <cuisinetext>
```

Comment:
Ignore `cuisinetext`, will remove next time the class doesn't work dk why so had to use td.even to change the hover for the cuisine